index.js

```diff
@@ -38,5 +24,6 @@ function monitorCtrlC(cb) {
     }
 }
 
-module.exports = monitorCtrlC;
-module.exports.defaultCtrlCHandler = defaultCtrlCHandler;
+module.exports = {
+    monitorCtrlC: monitorCtrlC
```

Comment:
Please also export default handler (as a peer of `monitorCtrlC`), in case a user of the library wants to just extend the default behaviour (i.e. create a function that composes their code and the default handler).